R/addins.R

```diff
@@ -1,8 +1,112 @@
-read_addin <- function(...) {
-  path <- fs::path_package("rhino", "rstudio", "addins", ...)
-  readChar(path, file.info(path)$size)
+run_background <- function(file) {
+  path <- fs::path_package("rhino", "rstudio", "addins", file)
+  rstudioapi::jobRunScript(path, workingDir = rstudioapi::getActiveProject())
 }
 
 addin_module <- function() {
-  rstudioapi::insertText(read_addin("module.R"))
+  module_path <- fs::path_package("rhino", "rstudio", "addins", "module.R")
+  file_path <- rstudioapi::selectFile(
+    caption = "Module name and location",
+    path = fs::path("app"),
+    filter = ".R",
+    label = "Save",
+    existing = FALSE
+  )
+  if (tools::file_ext(file_path) == "") {
+    file_path <- glue::glue("{file_path}.R")
+  }
+  file.copy(module_path, file_path)
```

Comment:
1. Please use `fs::file_copy` (for the consistency)
2. In case a user selects an existing file, `selectFile` will ask if they want to overwrite it. If a user answers "Yes", the addin should allow that.

Reply:
No, this is not a behavior we want. Please check my comment in the recent round of the review.